sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -172,6 +175,12 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
       // before next vsync so it will be drawn, otherwise we will drop it
       expectedVsyncWakeTimeNs = expectedEndRenderTimeNs - ONE_MILLISECOND_NS
     }
+    fpsChangedListener?.let {
+      val currentTime = SystemClock.elapsedRealtimeNanos()
```

Comment:
Can we reuse `actualEndRenderTimeNs` here?

Reply:
Yep, indeed!